packages/publications/src/Models/PublicationType.php

```diff
@@ -57,6 +57,9 @@ class PublicationType implements SerializableContract
     /** The number of publications to show per paginated page. Set to 0 to disable pagination. */
     public int $pageSize = 0;
 
+    /** Generic array field which can be used to store additional data as needed. */
+    public array $meta = [];
```

Comment:
What do you think about renaming this (and accessors) to `$metadata`?

```suggestion
    public array $metadata = [];
```

Reply:
My personal view is that since every class variable holds data, meta"DATA" is sort of superfluous. But if you want me to make this change, let me know and I'll do it.

Reply:
I see your point, for me it feels that "meta" is an abbreviation of "metadata" in this case. So as you are not opposed to the idea I think we should use "metadata". It looks a bit "cut off" IMO.